#331:  Use rounded_response_time for min/max/total response times

jude opened the pull request:
I was going to open a new ticket for these changes, but saw #331 already existed with almost the same changes that ended up working for me.  (I also use the rounded_response_time when computing the total response time)

I also added a few unit tests to cover the rounding behavior a little, and added a test_suite line to setup.py, so that 'python setup.py test' will find and run the tests.

heyman commented:
I agree that it's confusing that you might get 99 percentile that is larger than 100% (as discussed in #331). 

However, I don't think we should round the `total_response_time` as it's used to calculate the average response time. Rounding the min and max should be okay, but I'm also thinking that maybe it's even better to change so that we use `self.get_response_time_percentile()` for the 100% value as well (at this row: https://github.com/jude/locust/blob/a65de0fddc9bb6f4ca027325cbf510cfdbef7839/locust/stats.py#L358), and not rounding the min/max.

The rounding of the response times are a necessary evil that drastically shrinks the amount of data that needs to be sent from the worker nodes to the master. But that's not really needed for min and max.

jude commented:
Cool.  I've updated the PR as per the comment above.

I also updated the Vagrantfile to use xenial instead of precise, since it looks like precise has EOL'd since the the PR was opened.

heyman commented:
Great, thanks!
